README.md

```diff
@@ -45,6 +50,16 @@ Node 4.4.x and NPM 3.9.x (and above) are **required**. We suggest using [nvm](ht
 
 6. Navigate to [http://localhost:4200](http://localhost:4200)
 
+### Development with Docker
+
+To start a development server run
+
+```sh
+docker-compose up -d
+docker-compose exec toolchain /bin/bash # This opens bash inside of the docker container
```

Comment:
Name won't be `toolchain`, if it's the first DCOS-UI container will be `projectName_serviceName_#` so most likely `dcosui_toolchain_1`

Reply:
I get where you are coming from and with docker exec you are 💯 right. With docker-compose exec you specify the service name, that's why this command works